niflow_manager/cli/build.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+import subprocess as sp
+import tempfile, os, json
+from pathlib import Path
+from copy import deepcopy
+import yaml
+import click
+import neurodocker as ndr
+
+# default setting for specific neurodocker keys,
+# this value will not have to be set in the parameters.yml, but can be overwritten
+DEFAULT_INSTRUCTIONS = {"miniconda": {"create_env": "testkraken", "activate": True}}
```

Comment:
why is the build environment called `testkraken`?

Reply:
don't know, will change. `niflows` will be fine?

Reply:
> `niflows` will be fine?

yes